src/redisearch_api.c

```diff
@@ -267,9 +267,15 @@ int RediSearch_DeleteDocument(RefManager* rm, const void* docKey, size_t len) {
   if (id == 0) {
     rc = REDISMODULE_ERR;
   } else {
-    if (DocTable_Delete(&sp->docs, docKey, len)) {
```

Comment:
Do we still use `DocTable_Delete` or `DocTable_DeleteR` anywhere? Consider deleting if not